src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -717,10 +716,8 @@ void EnIk_SetupFrozen(EnIk* this) {
 }
 
 void EnIk_Frozen(EnIk* this, GlobalContext* globalCtx) {
-    if (this->counter != 0) {
-        this->counter--;
-    }
-    if (this->counter == 0) {
+    DECR(this->timer);
+    if (this->timer == 0) {
```

Comment:
```suggestion
    if (DECR(this->timer) == 0) {
```
is the usual way if it matches.

Reply:
I did initially try this and it did not match

Reply:
Ah, then it's questionable whether it's worth using DECR then, from a readability point of view.

Reply:
Right, I think I recall asking this and the consensus was to leave it in the expanded form